### PR TITLE
[ID] Update Project Loved

### PR DESCRIPTION
**Pull request by AncuL001**

### CODE REVIEW

File: wiki/Project_Loved/id.md [hunk 1]
```diff
@@ -48,4 +44,4 @@ Tidak ada aturan ketat yang mengatur map mana yang harus dinominasikan oleh kapt
 
 ## Informasi lebih lanjut
 
-Semua topik dan perencanaan untuk Project Loved dilakukan di server Discord [osu!dev channel](https://discord.gg/ppy) `#osu-loved`. Setiap pertanyaan, masalah, atau saran dapat dikirimkan ke sana.
+Semua penyelenggaraan dan perencanaan Project Loved dilakukan di channel `#osu-loved` dalam [server Discord osu!dev](https://discord.gg/ppy). Terdapat juga channel `#project-loved` di [server Discord Komunitas osu!](https://discord.gg/0Vxo9AsejDkGlk3H) Setiap pertanyaan, masalah, atau saran dapat dikirimkan ke sana.
```
Comment (hikikuman) [inline wiki/Project_Loved/id.md:47]:
```suggestion
Semua penyelenggaraan dan perencanaan Project Loved dilakukan di channel `#osu-loved` dalam [server Discord osu!dev](https://discord.gg/ppy). Terdapat juga channel `#project-loved` di [server Discord Komunitas osu!](https://discord.gg/0Vxo9AsejDkGlk3H). Setiap pertanyaan, masalah, atau saran dapat dikirimkan ke sana.
```
missing full stop.

File: wiki/Project_Loved/id.md [hunk 2]
```diff
@@ -13,7 +8,8 @@ Beatmap dikategorikan menjadi Loved setelah melewati proses nominasi dan pemungu
 
 Anggota Project Loved Team yang bergelar **kapten** secara teratur mencalonkan beatmap untuk mode tertentu. Hampir selalu, set yang dicalonkan berasal dari kiriman komunitas osu!. Formulir dan daftar pengajuan Loved saat ini dapat diakses melalui tautan berikut:
 
-- [Formulir osu!/osu!taiko/osu!catch](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
+- [Formulir osu!/osu!catch](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
+- [Formulir osu!taiko](https://docs.google.com/forms/d/e/1FAIpQLSclPWyjFByhHP45AtKD49y0RSl1TK5UOzD4dVdvjfJJQ1aCXQ/viewform)
 - [Formulir osu!mania](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform)
 - [Daftar osu!/osu!taiko/osu!catch](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit)
```
Comment (hikikuman) [inline wiki/Project_Loved/id.md:14]: `Daftar osu!/osu!taiko/osu!catch` should be changed to `Daftar osu!/osu!catch`. Don't forget to add `Daftar osu!taiko` to the list (also the link).

**Review by hikikuman — CHANGES_REQUESTED** [review]

File: wiki/Project_Loved/id.md (hunk 1 above)
Comment (AncuL001) [reply wiki/Project_Loved/id.md:47]: tfw

File: wiki/Project_Loved/id.md (hunk 2 above)
Comment (AncuL001) [reply wiki/Project_Loved/id.md]: fml

**Review by Nivalyx — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**